src/vue/forms/LimitsForm.vue

```diff
@@ -0,0 +1,319 @@
+  },
+  data: () => ({
+    form: {
+      dailyOut: '',
+      weeklyOut: '',
+      monthlyOut: '',
+      annualOut: '',
+      note: '',
+    },
+    selectedOpType: {},
+    opts: [],
+    STATS_OPERATION_TYPES,
+    OPERATION_TYPES_TRANSLATION_ID,
+    STATS_OPERATION_TYPES_KEY_NAMES,
+    formattedOpTypes: [],
+    isLimitsChanged: false,
+    formNoteMaxLength: 250,
+    config,
+  }),
+  validations () {
+    return {
+      form: {
+        dailyOut: {
+          decimal,
+          maxValueWrapper: maxValueWrapper(config.MAX_AMOUNT),
+          maxDecimalPoints: maxDecimalPoints(config.DECIMAL_POINTS),
+        },
+        weeklyOut: {
+          decimal,
+          maxValueWrapper: maxValueWrapper(config.MAX_AMOUNT),
+          maxDecimalPoints: maxDecimalPoints(config.DECIMAL_POINTS),
+        },
+        monthlyOut: {
+          decimal,
+          maxValueWrapper: maxValueWrapper(config.MAX_AMOUNT),
+          maxDecimalPoints: maxDecimalPoints(config.DECIMAL_POINTS),
+        },
+        annualOut: {
+          decimal,
+          maxValueWrapper: maxValueWrapper(config.MAX_AMOUNT),
+          maxDecimalPoints: maxDecimalPoints(config.DECIMAL_POINTS),
+        },
+        note: {
+          maxLength: maxLength(this.formNoteMaxLength),
+        },
+      },
+    }
+  },
+  computed: {
+    selectedLimitsByOpType () {
+      // eslint-disable-next-line
+      return this.limits[STATS_OPERATION_TYPES_KEY_NAMES[this.selectedOpType.value]]
+    },
+  },
+  created () {
+    this.formatOpTypes()
+    this.selectedOpType = this.formattedOpTypes[0]
+  },
+  methods: {
+    tryToSubmit () {
+      if (!this.isFormValid()) return
+      this.showFormConfirmation()
+    },
+    async submit () {
+      this.disableForm()
+      try {
+        await this.createRequest()
+        Bus.success('limits-form.request-successfully-created')
+      } catch (error) {
+        if (
+          error instanceof errors.TransactionError &&
+          // eslint-disable-next-line
+          error.includesOpCode(OPERATION_ERROR_CODES.opManageLimitsRequestReferenceDuplication)
+        ) {
+          Bus.error('limits-form.error-duplicate-request')
+        } else {
+          ErrorHandler.process(error)
+        }
+      }
+      this.enableForm()
+      this.hideFormConfirmation()
+      this.$emit(EVENTS.limitsChanged)
+    },
+    async createRequest () {
+      const asset = this.selectedLimitsByOpType.assetCode
+      const limits = {
+        annualOut: this.form.annualOut,
+        dailyOut: this.form.dailyOut,
+        monthlyOut: this.form.monthlyOut,
+        weeklyOut: this.form.weeklyOut,
+      }
+      const note = this.form.note
+      const requestType = LIMITS_REQUEST_TYPE.initial
+      const statsOpType = +this.selectedOpType.value
+      const operationType = STATS_OPERATION_TYPES_KEY_NAMES[statsOpType]
+
+      // eslint-disable-next-line
+      const operation = base.CreateManageLimitsRequestBuilder.createManageLimitsRequest({
```

Comment:
we can fold like the following
```
base
  .CreateManageLimitsRequestBuilder
  .createManageLimitsRequest({
```
to remove eslint-disable-next-line comment here